package.json

```diff
@@ -15,8 +15,12 @@
   "author": "",
   "license": "BSD",
   "dependencies": {
+    "axios": "^0.18.0",
     "copy-to-clipboard": "^3.0.8",
+    "deep-freeze-strict": "^1.1.1",
+    "history": "^4.7.2",
     "hjson": "^3.1.0",
+    "js-cookie": "^2.2.0",
```

Comment:
What's with the new dependencies here?

Reply:
Importing `ExecContext` type information causes the core `effects` module to be loaded, which in turn requires these peer dependencies. See the commit message at https://github.com/ai-labs-team/casium-devtools/pull/29/commits/b59db14e0b83e04a0127106719886ff34d732075.